@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -846,7 +838,7 @@ export class TiltViewClipPlanesEvaluator extends TopViewClipPlanesEvaluator {
         const r = EarthConstants.EQUATORIAL_RADIUS;
         const d = cameraToOrigin.length();
         let farPlane: number;
-        let farMax = mapView.lookAtDistance * this.farMaxRatio;
+        let farMax = mapView.targetDistance * this.farMaxRatio;
```

Comment:
I agree that `targetDistance` is a better name but I would stick to lookAtDistance to not break the API

Reply:
I have checked its only used internally and I have marked it as (@internal) too. I think we need to rename it ASAP before someone else will propagate this stuff.